components/script/dom/gpudevice.rs

```diff
@@ -45,8 +49,8 @@ use webgpu::wgpu::binding_model::{
     BindGroupEntry, BindGroupLayoutEntry, BindingResource, BindingType, BufferBinding,
 };
 use webgpu::wgt::{
-    BufferDescriptor, BufferUsage, ShaderStage, TextureComponentType, TextureFormat,
-    TextureViewDimension,
+    AddressMode, BufferDescriptor, BufferUsage, CompareFunction, FilterMode, SamplerDescriptor,
```

Comment:
FWIW, I generally just import `wgt` and accessing these types via `wgt::`. This would avoid confusion with similarly named types imported from other Servo parts, e.g. JS bindings.
In the end it's up to you and Servo import guidelines to make this call :)

Reply:
This is a good idea. I'll make the changes.